content/en/apps/guides/data/users-bulk-load.md

```diff
@@ -4,6 +4,8 @@ linkTitle: "Bulk Load Users"
 weight: 15
 description: >
   Notes for how to bulk load users
+aliases:
+    -    /core/guides/users-bulk-load
```

Comment:
Pedantic indentation parity fix:

```suggestion
  - /core/guides/users-bulk-load
```

Reply:
Thanks for noticing this! I had copied it from a page that had this formatting. I'll update both to be like the other 8.